pipelines/tekton-bundle-builder/kustomization.yaml

```diff
@@ -24,11 +24,9 @@ patches:
         value: $(params.path-context)
     # Remove tasks that assume a binary image
     - op: remove
-      path: /spec/tasks/9  # sbom-json-check
+      path: /spec/tasks/6  # deprecated-base-image-check
     - op: remove
-      path: /spec/tasks/5  # deprecated-base-image-check
-    - op: remove
-      path: /spec/tasks/4  # build-source-image
+      path: /spec/tasks/5  # build-source-image
```

Comment:
Can you include the comment block here that has the numbers for the `spec.tasks` here I would have expected both of these numbers to decrease if anything ... but actually remain unchanged because the json check happens afterwards.